cafebabel/templates/articles/macros.html

```diff
@@ -15,3 +15,12 @@
     Reading time: {{ article.body|reading_time }} min
   </p>
 {%- endmacro %}
+
+{%- macro tags_fields(tags, article=None) -%}
+  <div class=tags>
+    <label for=tag-1>Tags</label>
+    <ul class=tags-list></ul>
```

Comment:
@cafebabel-webmaster I wasn't sure of the purpose of that list, maybe useless now we have the datalist for suggestion?